libs/community/langchain_community/chains/pebblo_retrieval/base.py

```diff
@@ -98,10 +101,20 @@ def _call(
         """
         prompt_time = datetime.datetime.now().isoformat()
         PebbloRetrievalQA.set_prompt_sent(value=False)
-        _run_manager = run_manager or CallbackManagerForChainRun.get_noop_manager()
+        PebbloRetrievalQA.set_prompt_gov_sent(value=False)
         question = inputs[self.input_key]
+
         auth_context = inputs.get(self.auth_context_key, {})
         semantic_context = inputs.get(self.semantic_context_key, {})
+        is_valid_prompt = self._check_prompt_validity(question, semantic_context)
```

Comment:
Add the same logic to the async call method `_acall`.

Reply:
Added in _acall